cspell.json

```diff
@@ -0,0 +1,25 @@
+{
+    "language": "en",
+    "caseSensitive": false,
+    "dictionaries": [
+        "balena", "computerTerms",
+        "en_US",
+        "companies", "softwareTerms", "misc",
+        "javascript", "python", "typescript", "go",
+        "npm", "node", "html", "css", "cpp"
+    ],
+    "dictionaryDefinitions": [
+        {
+            "name": "balena",
+            "path": "./.cspell/balena-words.txt",
+            "addWords": true
+        },
+        {
+            "name": "computerTerms",
+            "path": "./.cspell/computer-terms.txt",
+            "addWords": true
+        }
+    ],
+    "exclude": ["*/*.svg"],
+    "ignorePaths": ["node_modules","CHANGELOG.md"]
```

Comment:
I wonder then where are the names coming from if changelog is already ignored.

Reply:
@zoobot Can you point to those examples?

Reply:
If we don't have an example then I recommend removing the names from the dictionary, also if you can answer the questions in my primary review comment https://github.com/balena-io/docs/pull/2417#pullrequestreview-1155904613